game-core/src/main/java/games/strategy/engine/config/client/LobbyServerPropertiesFetcher.java

```diff
@@ -67,11 +69,6 @@ public LobbyServerProperties fetchLobbyServerProperties() {
       return Optional.of(new LobbyServerProperties(testLobbyHostSetting.value(), testLobbyPortSetting.intValue()));
     }
 
-    if (testLobbyHostSetting.isSet() || testLobbyPortSetting.isSet()) {
-      ClientLogger.logQuietly("Ignoring lobby server override settings. "
-          + "You must override the lobby host and port settings simultaneously.");
-    }
```

Comment:
> I would recommend logging it at least at FINE level so it will be visible if we enable enhanced tracing

That would imply the game has already crapped out and a user is frustrated, sought help, and we tell them to turn on logging. It just seems after the fact. If this is a problem then not allowing the save to happen while the two components are not updated would be more appropriate. 

I think the alternative is to make this a 'loud' logging. The 'quiet' logging is also problematic as many issues have gone unnoticed quietly in the logs when they are mis-logged. Given we have a thick client game-app, basically we either handle the errors or abort/error. If this were server code I would be inclined to leave quiet logging, on client side I don't think it's worth anything in the end.